website/addons/dryad/routes.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+from framework.routing import Rule, json_renderer
+from website.addons.dryad import views
+from website.routes import OsfWebRenderer
+
+TEMPLATE_DIR = '../addons/dryad/templates/'
+
+
+page_routes = {'rules':
```

Comment:
Sure thing. Do you want this in the commit message? I can briefly write up what we've got right now in the PR messages.